presto-main/src/main/java/io/prestosql/SystemSessionProperties.java

```diff
@@ -121,6 +121,7 @@
     public static final String QUERY_MAX_TOTAL_MEMORY_PER_NODE = "query_max_total_memory_per_node";
     public static final String DYNAMIC_FILTERING_MAX_PER_DRIVER_ROW_COUNT = "dynamic_filtering_max_per_driver_row_count";
     public static final String DYNAMIC_FILTERING_MAX_PER_DRIVER_SIZE = "dynamic_filtering_max_per_driver_size";
+    public static final String DYNAMIC_FILTERING_PER_DRIVER_MIN_MAX_LIMIT = "dynamic_filtering_per_driver_min_max_limit";
```

Comment:
rename to `dynamic_filtering_min_max_limit_per_driver`

Reply:
renamed